### PR TITLE
Tableau- Getting Started by Creating a First View

### PR DESCRIPTION
**Pull request by KlippL**
New section added. Information to help start making a first visualization.

Issue #753

### CODE REVIEW

File: 02-fundamentals.Rmd [hunk 1]
```diff
@@ -614,6 +614,33 @@ Joins and Data Blending work when it is required to append columns from one tabl
...
+Pages                | The pages shelf allows for a series of separate page views with a common axis to better analyze how a certain *measure* affects the rest of the data. When a field is added to the pages shelf a control panel is added to the right side of the view to move through the pages (either manually or automatically).
+Measure Values       | At the bottom of the *measures* data pane is a special field that contains all measures of the data collected into one field. 
+
+
+**Calculation Type** | **Description**
+---------------------|-------------------------------------------------------------------------------------------------------------------
+**Dimention**        | Qualitative or categorical data that would normally appear as column headers for rows of data that normally defines the granularity that shows in the view, i.e. customer name or order date.
```
Comment (pragyansharma24) [inline 02-fundamentals.Rmd]: Hi Laura, 
Great addition.
I think the spelling of Dimension is incorrect. Please check. 
Thanks.

Comment (KlippL) [reply 02-fundamentals.Rmd]: Thanks for catching that @pragyansharma24, spelling errors have now been changed.

**Review by phoenixren — COMMENTED** [review]
This is helpful. Thank you.

**Comment by KlippL** [comment]
Thanks @phoenixren would you mind reviewing it so that I can merge it?

**Review by bharatimalik — APPROVED** [review]
Great reference guide for Tableau!

**Pull request merged**